Dockerfile

```diff
@@ -1,12 +1,17 @@
 FROM python:3.8-slim-buster
-MAINTAINER Marcin Sztolcman <marcin@urzenia.net>
+LABEL maintainer="Marcin Sztolcman <marcin@urzenia.net>"
 
+RUN python3 -m pip install pipenv
 RUN useradd --create-home sendria
-WORKDIR /home/sendria
 USER sendria
-RUN python3 -m pip install --user sendria==2.1.0
+COPY --chown=sendria:sendria . /home/sendria/sendria
+WORKDIR /home/sendria/sendria
+RUN pipenv lock --requirements > requirements.txt
+RUN python3 -m pip install --user -r requirements.txt
+RUN /home/sendria/.local/bin/webassets -m sendria.build_assets build
+RUN python3 setup.py install --user
```

Comment:
Maybe it's worth to install sendria as editable for local hacking? Wouldn't it be more elastic? Then just restarting container should be enough to see local changes instead of building from beginning?

Reply:
While that would probably be the best for local hacking, I have my qualms with this method, mainly the fact that it causes discrepancy between dev and prod environments. 
But it's easy enough to do so I can edit the PR to go in that direction if you want.
That being said, that would mean you would have multiple Dockerfiles (one for the local development, let's say Dockerfile, and the other for building the release image, Dockerfile.prod).